miscmodule/showroll_command.go

```diff
@@ -0,0 +1,126 @@
+package miscmodule
+
+import (
+	"math"
+	"math/rand"
+	"regexp"
+	"strconv"
+	"strings"
+
+	bot "../sweetiebot"
+	"github.com/blackhole12/discordgo"
+)
+
+var diceregex = regexp.MustCompile("[0-9]*d[0-9]+")
+
+type showrollCommand struct {
+}
+
+func (c *showrollCommand) Info() *bot.CommandInfo {
+	return &bot.CommandInfo{
+		Name:              "Showroll",
+		Usage:             "Evaluates a dice expression.",
+		ServerIndependent: true,
+	}
+}
+func (c *showrollCommand) eval(args []string, index *int, info *bot.GuildInfo) string {
+	var s string
+	for *index < len(args) {
+		s += value(args, index) + "\n"
+	}
+	return s
+}
+func (c *showrollCommand) value(args []string, index *int, info *bot.GuildInfo) string {
```

Comment:
Because this is intended as a server-independent command, instead of passing the entire GuildInfo object here, just pass the actual CommandPrefix so it's obvious that no other part of the GuildInfo object is used.